src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/actionpostponementrequest/ActionPostponementRequestAboutToSubmitHandlerTest.java

```diff
@@ -272,6 +272,10 @@ public void givenNonSAndLCase_shouldNoMessages() {
         PreSubmitCallbackResponse<SscsCaseData> response =
             handler.handle(ABOUT_TO_SUBMIT, callback, USER_AUTHORISATION);
 
+        assertThat(response.getErrors())
+            .hasSize(1)
```

Comment:
Question: do we care that there is only one error? Or we just want to ensure that there is "at least" the error we are looking for?